src/views/Nft/market/Profile/components/SubMenu.tsx

```diff
@@ -19,15 +10,15 @@ const SubMenuComponent: React.FC = () => {
   const ItemsConfig = [
     {
       label: t('Items'),
-      href: `${nftsBaseUrl}/profile`,
+      href: '/nft/market/profile',
```

Comment:
Any reason why you don't use `nftsBaseUrl` ?  Most likely the url will change

Reply:
Just a revert from the rebase.